src/main/kotlin/lotto/LottoMachine.kt

```diff
@@ -0,0 +1,35 @@
+package lotto
+
+class LottoMachine(
+    val amountOfMoney: Int,
+    val lastWeekWinningNumbers: List<Int>,
+    val bonusNumber: Int,
+) {
+    val amountOfTicket: Int = amountOfMoney / LottoTicket.COST_OF_TICKET
```

Comment:
(If time allows, later...)
How about avoiding internal variables beyond those in the primary constructor?
Returning results directly from methods each time can eliminate side effects, clarify the object’s responsibilities, and reduce complexity. 🙂